stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -730,6 +730,32 @@ public func withUnsafeThrowingContinuation<T>(
   }
 }
 
+// HACK: a version of withUnsafeContinuation that uses the unsafe
+// @_unsafeInheritExecutor.
+@available(SwiftStdlib 5.1, *)
+@_alwaysEmitIntoClient
+@_unsafeInheritExecutor
+public func _unsafeInheritExecutor_withUnsafeContinuation<T>(
+  _ fn: (UnsafeContinuation<T, Never>) -> Void
+) async -> T {
```

Comment:
This should be `sending T`, shouldn't it? At the very least, its now inconsistent between throwing and not throwing which seems off. 

And note that both our "good" APIs are `sending`


```
@_alwaysEmitIntoClient
public func withUnsafeContinuation<T>(
  isolation: isolated (any Actor)? = #isolation,
  _ fn: (UnsafeContinuation<T, Never>) -> Void
) async -> sending T {

@_alwaysEmitIntoClient
public func withUnsafeThrowingContinuation<T>(
  isolation: isolated (any Actor)? = #isolation,
  _ fn: (UnsafeContinuation<T, Error>) -> Void
) async throws -> sending T {
```

Reply:
Good catch, thank you!